README.md

```diff
@@ -7,620 +7,799 @@
-As a facet of our React application, we want to provide users with the option to
-see a list of our favorite soils. Currently, instead of sharing a link to a list
-of our favorite soils, we can only provide a link to our "Love for soils"
-homepage. Following which, users are required to interact with our application
-to see a favorite soil list.
+As developers of this website, we want to provide users with the option to see a
+list of our favorite soils. Currently, instead of sharing a link to that list,
+we can only provide a link to our "Love for soils" homepage. Users are then
+required to interact with our application to see the favorite soil list.
 
 Because our personal opinion on the best soils is so important, we want to
-provide users with the opportunity to go straight to this list of the favorite
-soils view with a URL at `www.loveforsoils.com/favorites`. Enter **React
-Router**: a routing library for **React** that allows us to link to specific
-URLs and conditionally render components depending on which URL is displayed.
-
-React Router is a collection of navigational components and custom hooks that
-are implemented using declarative programming and [compose with][composition]
-the components in your application. Whether you want to have bookmark-able URLs
-for your web app, or a composable way to navigate in React Native, React Router
-works wherever React is rendering — so take your pick!
-
-[composition]: https://reactjs.org/docs/composition-vs-inheritance.html
+provide users with the opportunity to go straight to this list of our favorite
+soils by using the URL `www.loveforsoils.com/favorites`. Enter **React Router**:
+a routing library for **React** that allows us to link to specific URLs and
+conditionally render components depending on which URL is displayed.
+
+As mentioned in the previous reading, **React Router** enables _client-side
+routing_ which allows us to render different portions of our webpage using the
+[browser's History
+API](https://reactrouter.com/en/main/start/concepts#history-and-locations)
+instead of making requests to our server for a new webpage. Instead, our browser
+renders a new component, and our client-side JavaScript requests any data we
+want to display in that component. This is essential for routing in any React
+application, as we only have a single HTML file to serve - that's the nature of
+an SPA.
+
+>**Note** Some web development frameworks that use and expand upon React, like
+>`Next.js`, handling routing and rendering a little differently, and start to
+>depart from React's traditional client-side SPA implementation. `Remix`, which
+>is the React based framework that maintains React Router, still uses React
+>Router to handle its routing, but the underlying rendering and re-rendering of
+>the application is handled a little differently. When we build a React project
+>outside of these additional frameworks, React Router will perform client-side
+>routing in the way we described above.
```

Comment:
I was just throwing it in for overall context. I was partially motivated by the fact that React now no longer suggests making plain React projects, and instead guides people to use these other two frameworks. But I agree that it's unnecessary - if we're looking to trim material, this would be high on my list.

Reply:
Yeah I get that. Alvee submitted a PR with some language about using frameworks in the Create React App lesson so it'll be covered once that's merged in. I think it's fine to remove it from this lesson